selenium/src/test/java/au/org/emii/portal/tests/BrowserStackTest.java

```diff
@@ -0,0 +1,19 @@
+package au.org.emii.portal.tests;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+public class BrowserStackTest extends BaseTest {
+
+    @Test(invocationCount = invocationCount, threadPoolSize = threadPoolSize)
+    public void BrowserStackTest() {
```

Comment:
Actually its named like a constructor which is confusing - the above is actually a method.